packages/fuels-test-helpers/src/lib.rs

```diff
@@ -21,15 +24,58 @@ mod wallets_config;
 pub use signers::*;
 pub use wallets_config::*;
 
-pub fn setup_coins(owner: Address, num_coins: u64, amount: u64) -> Vec<(UtxoId, Coin)> {
+/// Create a vector of `num_asset`*`coins_per_asset` UTXOs and a vector of the unique corresponding
+/// asset IDs. `AssetId`. Each UTXO (=coin) contains `amount_per_coin` amount of a random asset. The
+/// output of this function can be used with `setup_test_client` to get a client with some
+/// pre-existing coins, with `num_asset` different asset ids. Note that one of the assets is the
+/// native asset to pay for gas.
+pub fn setup_multiple_assets_coins(
+    owner: Address,
+    num_asset: usize,
+    coins_per_asset: u64,
+    amount_per_coin: u64,
+) -> (Vec<(UtxoId, Coin)>, Vec<AssetId>) {
+    let mut rng = rand::thread_rng();
+    let mut coins = (1..num_asset)
```

Comment:
This is a bit confusing for me, I thought initially it was an off-by-one error, as `(1..num_asset)` produces `num_asset - 1` items. I see that `setup_single_asset_coins` is added later, but it's not immediately apparent.

Something like `(0..(num_asset - 1))`, albeit uglier, conveys the intention better IMO.